File: 0660-remove-9/0660-remove-9.cpp
class Solution {
public:
    int newInteger(int n) {
        vector<int> s;
        while(n) {
            s.push_back(n % 9);
            n /= 9;
        }
        int ans = 0;
        for(int i = s.size() - 1; i >= 0; --i) {
            ans = ans * 10 + s[i];
        }
        
        return ans;
    }
};

File: 0660-remove-9/0660-remove-9.js
/**
 * @param {number} n
 * @return {number}
 */
var newInteger = function(n) {
    let s = [];
    while (n) {
        s.push(n % 9);
        n = Math.floor(n / 9);
    }
    let ans = 0;
    for (let i = s.length - 1; i >= 0; --i) {
        ans = ans * 10 + s[i];
    }
    return ans;
};